### PR TITLE
chore(docs/examples): Updating Tempo X Node.js

### PR DESCRIPTION
**Pull request by xoscar**
This PR updates the quick start node.js and tempo example and recipe

## Changes

- Updates recipe to use new template

## Fixes

- https://github.com/kubeshop/tracetest-cloud/issues/807

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: docs/docs/examples-tutorials/recipes/running-tracetest-with-grafana-cloud-tempo-pokeshop.mdx [hunk 1]
```diff
@@ -64,11 +62,10 @@ cd tracetest/examples/tracetest-grafana-cloud-tempo-pokeshop
 Follow these instructions to run the quick start:
 
 1. Copy the `.env.template` file to `.env`.
-2. Log into the [Tracetest app](https://app.tracetest.io/).
-3. Fill out the [token](https://docs.tracetest.io/concepts/environment-tokens) and [API key](https://docs.tracetest.io/concepts/agent) details by editing your `.env` file. You can find these values in the Settings area for your environment.
-4. Run `docker compose up -d`.
-5. This example is configured to use the Grafana Tempo Tracing Backend. Ensure the environment you're using to run this example is configured to use the Grafana Tempo Tracing Backend by clicking on Settings, Tracing Backend, Grafana Tempo, Save. Or, use the CLI as explained below.
-6. Run tests from the Tracetest Web UI by accessing the app with the URL `http://demo-api:8081/pokemon/import`.
+2. Fill out the [TRACETEST_TOKEN and ENVIRONMENT_ID](https://app.tracetest.io/retrieve-token) details by editing your `.env` file.
+3. Fill out the [GRAFANA_AUTH_READ_HASH and GRAFANA_AUTH_WRITE_HASH](https://grafana.com/blog/2021/04/13/how-to-send-traces-to-grafana-clouds-tempo-service-with-opentelemetry-collector/) deatails from your Tempo setup
```
Comment (jfermi) [inline docs/docs/examples-tutorials/recipes/running-tracetest-with-grafana-cloud-tempo-pokeshop.mdx:66]:
```suggestion
3. Fill out the [GRAFANA_AUTH_READ_HASH and GRAFANA_AUTH_WRITE_HASH](https://grafana.com/blog/2021/04/13/how-to-send-traces-to-grafana-clouds-tempo-service-with-opentelemetry-collector/) deatails from your Tempo setup.
```

File: docs/docs/examples-tutorials/recipes/running-tracetest-with-grafana-tempo-pokeshop.mdx [hunk 2]
```diff
@@ -32,23 +32,59 @@ This is a simple quick start guide on how to configure a fully instrumented API
...
+```
+
+Follow these instructions to run the quick start:
+
+1. Copy the `.env.template` file to `.env`.
+2. Fill out the [TRACETEST_TOKEN and ENVIRONMENT_ID](https://app.tracetest.io/retrieve-token) details by editing your `.env` file.
+3. Run `docker compose run tracetest-run`.
+4. Follow the links in the output to view the test results.
+
+Follow along with the sections below for an in detail breakdown of what the example you just ran did and how it works.
```
Comment (jfermi) [inline docs/docs/examples-tutorials/recipes/running-tracetest-with-grafana-tempo-pokeshop.mdx:60]:
```suggestion
Follow the sections below for a detailed breakdown of what the example you just ran did and how it works.
```

File: docs/docs/examples-tutorials/recipes/running-tracetest-with-grafana-tempo-pokeshop.mdx [hunk 3]
```diff
@@ -32,23 +32,59 @@ This is a simple quick start guide on how to configure a fully instrumented API
...
 The project is built with Docker Compose.
 
 ### 1. Tracetest
 
-The `collector.config.yaml` file, `tracetest.provision.yaml`, and `tracetest.config.yaml` in the root directory are for the configuring Tracetest and it's OpenTelemetry Collector.
+The project contains [Tracetest Agent](/getting-started/installation#install-the-tracetest-agent), [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main), and a Node.js app.
+
+The `docker-compose.yaml` file in the root directory of the quick start runs the Node.js app and the [Tracetest Agent](/concepts/agent) setup.
+
+Configure the `.env` like shown below.
```
Comment (jfermi) [inline docs/docs/examples-tutorials/recipes/running-tracetest-with-grafana-tempo-pokeshop.mdx:72]:
```suggestion
Configure the `.env` as shown below.
```

**Review by jfermi — COMMENTED** [review]
Some editing suggestions.

**Review by adnanrahic — APPROVED** [review]

**Pull request merged**